### PR TITLE
feat: Sort by default according to the first column

### LINKED ISSUE #1265: [autogrid] Default sorting
An `<AutoGrid>` should be sorted by default. As we currently have no info about what would be a good column to sort on, we should use the first one

### PR DESCRIPTION
**Pull request by Artur-**
Fixes #1265

### CODE REVIEW

File: packages/ts/react-grid/src/autogrid.tsx [hunk 1]
```diff
@@ -182,7 +185,9 @@ export function AutoGrid<TItem>({
   useEffect(() => {
     // Sets the data provider, should be done only once
     const grid = ref.current!;
-    grid.dataProvider = createDataProvider(grid, service, dataProviderFilter);
+    setTimeout(() => {
```
Comment (sissbruecker) [inline packages/ts/react-grid/src/autogrid.tsx:188]: Could add a comment to explain the timeout.

File: packages/ts/react-grid/test/autogrid.spec.tsx [hunk 2]
```diff
@@ -38,52 +45,72 @@ async function assertColumns(result: RenderResult, ...ids: string[]) {
...
+      result.rerender(<TestAutoGrid />);
+      expect(getSortOrder(grid)).to.eql({ path: 'lastName', direction: 'desc' });
+    });
+    it('creates sortable columns', async () => {
+      const result = render(<TestAutoGrid />);
+      const grid: GridElement = result.container.querySelector('vaadin-grid')!;
+      await nextFrame();
+      await nextFrame();
+      const firstNameSorter = getHeaderCellContent(grid, 0, 0).firstElementChild as HTMLElement;
       firstNameSorter.click();
```
Comment (sissbruecker) [inline packages/ts/react-grid/test/autogrid.spec.tsx:85]: Can reuse `sortGrid` here.

Comment (Artur-) [reply packages/ts/react-grid/test/autogrid.spec.tsx]: Done

File: packages/ts/react-grid/src/autogrid.tsx (hunk 1 above)
Comment (Artur-) [reply packages/ts/react-grid/src/autogrid.tsx:188]: Added

**Review by sissbruecker — APPROVED** [review]

**Pull request merged**